packages/react-router/lib/router/router.ts

```diff
@@ -3658,7 +3670,9 @@ export function createStaticHandler(
                   dataStrategy || null,
                   skipLoaderErrorBubbling === true,
                   null,
-                  filterMatchesToLoad || null,
+                  "filterMatchesToLoad" in opts
+                    ? (opts.filterMatchesToLoad ?? null)
+                    : null,
```

Comment:
I think we want to fall back on the original value here if they didn't pass an override?

```suggestion
                    : (filterMatchesToLoad ?? null),
```

I should go through and clean up some of the `null`/`undefined` stuff in here one of these days 😬

Reply:
actually we need it for the first half of the ternary in case they specified an undefined key - they shouldn't, but the types would allow it.  Maybe this is easier?

```
opts.filterMatchesToLoad ?? filterMatchesToLoad ?? null,
```